core/kazoo_attachments/src/kz_att_onedrive.erl

```diff
@@ -143,8 +146,10 @@ do_fetch_attachment({'ok', #{'token' := #{'authorization' := Authorization}}},
                     DriveId, ContentId, HandlerProps, DbName, DocId, AName) ->
     Headers = [{<<"Authorization">>, Authorization}],
     Url = ?DRV_FETCH_URL(DriveId, ContentId),
-    lager:debug("getting ~p", [Url]),
-    case kz_http:get(Url, Headers, ?GRAPH_HTTP_OPTIONS) of
+    lager:debug("getting ~p : ~p", [Url, Headers]),
+    %% case kz_http:get(Url, Headers, ?GRAPH_HTTP_OPTIONS) of
+    %% SEEMS HTTPC IS BROKEN IN OTP 21.2.5
```

Comment:
What's the issue you found?

Reply:
not conclusive, but it fails consistently, needs further debugging, (maybe related to size of header content , url encoding, [insert yours here]). same thing in `aws` which was using `httpc` by default and the upgrade of `erlcloud` worked well